packages/features/ee/users/components/UsersTable.tsx

```diff
@@ -134,7 +134,7 @@ function UsersTableBare() {
                       size="md"
                       alt={`Avatar of ${user.username || "Nameless"}`}
                       gravatarFallbackMd5=""
-                      imageSrc={`${WEBAPP_URL}/${user.username}/avatar.png`}
+                      imageSrc={`${WEBAPP_URL}/${user.username}/avatar.png?orgId=${user.organizationId}`}
```

Comment:
Instead of getting org domain here for every user separately, I have started passing orgId directly and thus avatar endpoint has the org context in this way.